src/tree_graphviz.ml

```diff
@@ -135,3 +135,21 @@ let to_dot : string -> sym -> unit = fun fname s ->
   let oc = open_out fname in
   output_tree (Format.formatter_of_out_channel oc) tree;
   close_out oc
+
+(** [pp o m] prints matrix [m] to out channel [o]. *)
+let pp_matrix : Tree.CM.t pp = fun oc m ->
+  let open Tree.CM in
```

Comment:
This isn't related at all with graphviz at all, we might change the name of the file

Reply:
And moreover, I mainly used this function to debug and see matrices in the `compile` function, so I definitively think it should rather be in `tree.ml` (and by the way, this `pp` is not so pretty, I'll rework on it some day)

Reply:
Yeah, that does not really matter. Note that in one of my latest patch I actually added a note saying that these functions are useful only for debugging. This is still graphiviz generation right? Of course it is not used to print trees, but it still make sense to have these in that file. And I'm not too worried about the cleanness of that file, this code is not critical.

Reply:
Hmm I see, well they are indeed not used in nominal use, unless we make a flag to show the matrices, so I understand.  On the other hand, they are not related to graphviz, these pp things are for console output

Reply:
OK. I would prefer to leave these printing functions in the `src/tree_graphviz.ml` file though, but you are right that the name of the file is not great if we do that. Maybe we should rename the file into `src/tree_output.ml`?

Reply:
Actually I'm fine with the function remaining in `src/tree.ml`, but if you choose to keep that then also move back the function `pp_cond_pool` to the module `CP` in `src/tree.ml`.

Reply:
Well alleviating `tree.ml` is a good idea, but it is a bit of a shame not being able to use `pp_matrix` where it is the more likely to be needed, I don't really know which one is best

Reply:
Well let's put it in a `tree_output.ml`, we'll see on the long run what is best

Reply:
Oh I did not think about that, you are right. Maybe we should put the printing functions in the `CP` and `CM` modules then, and only keep the graphviz stuff in the other module (so its previous name was perfect).